### PR TITLE
Us 259168

### PR DESCRIPTION
**Pull request by DomanskiJakub**

### CODE REVIEW

File: android/core/src/main/java/com/pega/vehicledamagemodeling/VehicleDamageModeling.java [hunk 1]
```diff
@@ -87,30 +89,33 @@ public void render () {
...
+        Vector3 temporaryCamPosition = cam.position;
 
-    @Override
-    public void pause() {
-    }
+        if (carShouldBeRotated) {
+            cam = new PerspectiveCamera(60, height, width);
+        } else {
+            cam = new PerspectiveCamera(60, width, height);
+        }
```
Comment (SzymonSmykala) [inline android/core/src/main/java/com/pega/vehicledamagemodeling/VehicleDamageModeling.java]: It should be in the factory class.

File: android/core/src/main/java/com/pega/vehicledamagemodeling/VehicleDamageModeling.java [hunk 2]
```diff
@@ -40,6 +41,7 @@
     private Environment environment;
     private boolean loading;
     private static final String MODEL_FILE_NAME = "model.2.0.obj";
+    private boolean carShouldBeRotated = false;
```
Comment (SzymonSmykala) [inline android/core/src/main/java/com/pega/vehicledamagemodeling/VehicleDamageModeling.java]: private boolean fields should be above final fields

File: android/core/src/main/java/com/pega/vehicledamagemodeling/VehicleDamageModeling.java (hunk 1 above)
Comment (DomanskiJakub) [reply android/core/src/main/java/com/pega/vehicledamagemodeling/VehicleDamageModeling.java]: Problem is no longer present after refactoring.

**Comment by MartinJurczyk** [comment]
correct

**Pull request merged**